docs/examples/use_cases/tensorflow/efficientdet/pipeline/dali/fn_pipeline.py

```diff
@@ -0,0 +1,254 @@
+# Copyright 2021 Kacper Kluk, Paweł Anikiel, Jagoda Kamińska. All Rights Reserved.
```

Comment:
General comment about this file:
why the name `fn_pipeline`? The functional API is now the main API. A more adequate name would be `efficientdet_pipeline` or `ed_pipeline` or maybe just `data_pipeline`.

Reply:
Good point, it's a remnant from the time when we had pipelines in both APIs.